### ISSUE TITLE
Referenced hxml is ignored when building with openfl/lime

### ISSUE THREAD

**Issue by kevinresol** [lix-pm/lix.client#129]
project.xml:
```xml
<?xml version="1.0" encoding="utf-8"?>
<project>
	
	<meta title="Test" package="com.kevinresol.test" version="1.0.0" company="kevinresol" />
	<app main="Main" path="bin" file="Test" />
	<window background="#000000"/>
	
	<source path="src" />
	
	<haxelib name="openfl" />
	<haxelib name="custom-lib" />
	
</project>
```

suppose `haxe_libraries/custom-lib.hxml` looks like so:

```hxml
another.hxml
-cp ../custom-lib/src
```

Here, `another.hxml` will be completely ignored during compilation.

@Gama11 I think you are the one who uses both lix and openfl. Have you experienced this?

**Comment by Gama11** [comment]
> @Gama11 I think you are the one who uses both lix and openfl. Have you experienced this?

I don't use lix _with_ OpenFL. I think Lime isn't really compatible yet. https://github.com/haxelime/lime/pull/1381

**Comment by back2dos** [comment]
Hmm. What does `haxelib path custom-lib` return for you? Or does lime/openfl not use haxelib?

**Comment by Gama11** [comment]
It does.

**Comment by back2dos** [comment]
Seems I broke the recursion through hxmls in haxeshim.

**Comment by back2dos** [comment]
Fixed in 15.8.15.
